web/assets/javascripts/metrics.js

```diff
@@ -0,0 +1,41 @@
+function init() {
+  const ctx = document.getElementById("metrics-chart");
+  const colors = JSON.parse(ctx.getAttribute("data-colors"));
+  const labels = JSON.parse(ctx.getAttribute("data-labels"));
+  const datasets = Object.entries(
+    JSON.parse(ctx.getAttribute("data-series"))
+  ).map(([k, v], i) => {
+    return {
+      label: k,
+      data: v,
+      borderColor: colors[i],
+      backgroundColor: colors[i],
+      borderWidth: 2,
+      pointRadius: 2,
+    };
+  });
+  const options = {
+    aspectRatio: 4,
+    scales: {
+      y: {
+        beginAtZero: true,
+      },
+    },
+    interaction: {
+      mode: "x",
+    },
+    plugins: {
+      legend: {
+        display: false,
+      },
+    },
+  };
+
+  const chart = new Chart(ctx, {
+    type: "line",
+    data: { labels: labels, datasets: datasets },
+    options: options,
+  });
+}
+
+document.addEventListener("DOMContentLoaded", init);
```

Comment:
The dashboard uses this [custom `ready` function](https://github.com/adamlogic/sidekiq/blob/3ab19ed514c10510e4e403067fcd61e5f2b48164/web/assets/javascripts/dashboard.js#L224-L227), but I don't know if it's necessary.

Reply:
I don't know either. If it's not needed, it's a bug.